scripts/rollup/wrappers.js

```diff
@@ -354,7 +354,9 @@ if (process.env.NODE_ENV !== "production") {
 ${source}
     return exports;
   };
-}`;
+  module.exports.default = module.exports;
```

Comment:
Hmm. Per https://github.com/rollup/rollup/pull/3710 , `interop: 'esModule'` just cares about the import and uses `.default`, and doesn't check for the existence of `__esModule` on the imported value.  So, I don't think that helps with the _internal_ usages.  But, I suppose it might help with external ones. I'll throw it in there.